lightning/src/onion_message/messenger.rs

```diff
@@ -630,8 +634,8 @@ where
 		) {
 			Ok(PeeledOnion::Receive(message, path_id, reply_path)) => {
 				log_trace!(self.logger,
-					"Received an onion message with path_id {:02x?} and {} reply_path",
-						path_id, if reply_path.is_some() { "a" } else { "no" });
+						"Received an onion message with path_id {:02x?} with message : {:?} and reply_path : {}", path_id, 
+						message, if reply_path.is_some() { "a" } else { "no" });
```

Comment:
Let's write this so that the article is in the right spot and the message is at the end:

```rust
log_trace!(
	 self.logger,
	"Received an onion message with path_id {:02x?} and {} reply_path: {:?}",
	 path_id, if reply_path.is_some() { "a" } else { "no" }, message
);
```

Will need to update the fuzz test accordingly.